docs/sources/installation/fedora.md

```diff
@@ -4,37 +4,32 @@ page_keywords: Docker, Docker documentation, Fedora, requirements, virtualbox, v
 
 # Fedora
 
-Docker is available in **Fedora 19 and later**. Please note that due to
-the current Docker limitations Docker is able to run only on the **64
+Docker is supported only on supported versions of fedora.
```

Comment:
Agreed, this is kind of confusing.  Seems like a tautology. Do you mean "Docker is only supported on the current version of Fedora"? Or maybe, "Docker is only supported on Fedora 19 or later", or...?